runtime/src/main/kotlin/konan/worker/Worker.kt

```diff
@@ -240,4 +235,12 @@ internal fun ThrowWorkerInvalidState(): Unit =
         throw IllegalStateException("Illegal transfer state")
 
 @ExportForCppRuntime
-internal fun WorkerLaunchpad(function: () -> Any?) = function()
+internal fun WorkerLaunchpad(function: () -> Any?): Any? {
+    try {
+        return function()
+    } catch (e: Throwable) {
+        println("worker thrown an exception: $e")
```

Comment:
Has the worker thrown this exception, or it was the thread sending the job to the worker?

Reply:
Done